vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/test/grid-connector-sorting.test.ts

```diff
@@ -0,0 +1,132 @@
+import { expect, fixtureSync, nextFrame } from '@open-wc/testing';
+import { init, getBodyCellContent, setRootItems } from './shared.js';
+import type { FlowGrid, Item } from './shared.js';
+import sinon from 'sinon';
+import { GridColumn } from '@vaadin/grid';
+import { GridSorter } from '@vaadin/grid/vaadin-grid-sorter.js';
+
+describe('grid connector - sorting', () => {
+  let grid: FlowGrid;
+  let columns: GridColumn<Item>[];
+  let sorters: GridSorter[];
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="name"></vaadin-grid-column>
+        <vaadin-grid-column path="price"></vaadin-grid-column>
+      </vaadin-grid>
+    `);
+
+    init(grid);
+
+    columns = [...grid.querySelectorAll('vaadin-grid-column')] as GridColumn<Item>[];
```

Comment:
You can avoid casts by adding a type to querySelectorAll: `grid.querySelectorAll<GridColumn<Item>>('vaadin-grid-column')`

Reply:
Good catch, thanks!